download/download.go

```diff
@@ -327,6 +330,15 @@ func downloadFile(uri, token, pubKeyBase64, filePath string) error {
 	if pubKeyBase64 != "" {
 		filePath += ".c4gh"
 	}
+
+	if *continueDownload {
+		if _, err := os.Stat(filePath); !errors.Is(err, os.ErrNotExist) {
+			fmt.Printf("Skipping download to %s, file already exists\n", filePath)
+
+			return nil
+		}
+	}
```

Comment:
```suggestion
	if _, err := os.Stat(filePath); !errors.Is(err, os.ErrNotExist) && *continueDownload {
		fmt.Printf("Skipping download of %s, file already exists\n", filePath)

		return nil
	}
```

It should be doable without the nested ifs.
Also should the message be to stdout or stderr?

Reply:
The nested if was a conscious choice because this way we spare the unnecessary calls to `os.Stat` for every file if `-continue` is false (imagine 50000 files for example). 

As for the second part, I see that we use a mixture of stderr and stdout for our messages. For example, `Upload` outputs files that are skipped to stdout whereas `decrypt` to stderr, the later has a `Warning:` prefix as well. So I am inclined to leave it as is and open an issue to harmonize all relevant messages.

Reply:
Fair enough.